ibridgesgui/popup_widgets.py

```diff
@@ -435,38 +451,38 @@ def close_window(self):
         """Close window while data transfer stays in progress."""
         if self.active_download:
             reply = QMessageBox.critical(
-                        self, "Message",
-                        "Do you want to close the window while the transfer continues?",
-                        QMessageBox.StandardButton.Yes,
-                        QMessageBox.StandardButton.No,
-                    )
+                self,
+                "Message",
+                "Do you want to close the window while the transfer continues?",
+                QMessageBox.StandardButton.Yes,
+                QMessageBox.StandardButton.No,
+            )
             if reply == QMessageBox.StandardButton.Yes:
                 self.active_download = False
         self.close()
 
     # pylint: disable=C0103
-    def closeEvent(self, evnt): # noqa
+    def closeEvent(self, evnt):  # noqa
         """Override close when download is in process."""
         if self.active_download:
             evnt.ignore()
 
-
     def irods_path_tree(self):
         """Expand the irods_path if it is a collection."""
         if self.irods_path.collection_exists():
             return "\n".join(
-                    [coll.name for coll in self.irods_path.collection.subcollections]\
-                    + [obj.name for obj in self.irods_path.collection.data_objects ])
+                [coll.name for coll in self.irods_path.collection.subcollections]
+                + [obj.name for obj in self.irods_path.collection.data_objects]
+            )
```

Comment:
I would split this up into several commands.

Reply:
I will keep it in mind. But for now, with all the other changes I would rather not like to change it yet. Next round?